jupyter_kernel_mgmt/subproc/manager.py

```diff
@@ -94,9 +80,9 @@ def kill(self):
                     raise
 
         # Block until the kernel terminates.
```

Comment:
Nitpick: block -> wait

Reply:
Also, I think the code here should call `await self.wait()` rather than `await self.kernel.wait()` since I'm seeing a PEP warning because popen wait returns an `int`.

Reply:
It should be the same, because `kernel` should be an asyncio process object, no?

Reply:
Yeah, just stepped through it.  This warning is pycharm getting confused.